Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -105,6 +105,15 @@ namespace Multiplayer
         "Upon receiving the information, the client will print the mismatch information to the game log. "
         "Provided for debugging during development, but you may want to mark false for release builds.");
 
+    AZ_CVAR(bool, bg_capturePhysicsTickMetric, true, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
+        "Should the Multiplayer gem record average physics tick time?");
+    AZ_CVAR(bool, bg_captureTransportMetrics, true, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
+        "Should the Multiplayer gem record transport metrics?");
+    AZ_CVAR(int, bg_captureTransportType, 0, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
```

Comment:
You should be able to use the protocol enum
```suggestion
    AZ_CVAR(AzNetworking::ProtocolType, bg_captureTransportType, AzNetworking::ProtocolType::Udp, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
```

Reply:
Done.